spec/Powertrain/Battery.vspec

```diff
@@ -12,6 +12,16 @@
 # For some vehicles the traction battery is referred to as the high voltage battery.
 #
 
+Id:
+  datatype: string
+  type: attribute
+  description: Battery Identification Number as assigned by OEM.
```

Comment:
What is this ID meant to represent? Some serial number? A location in the vehicle? A differentiator in case there are more than one batteries in the vehicle?

Reply:
Description could be improved, the intention is not to specify that this is battery "2" in the vehicle, but rather a (more or less) unique identifier of the battery instance mounted in the vehicle. E.g. serial number, UUID, part_number+serial_number or similar.

As far as I know there are no standard similar to Vehicle Identification Number (VIN) that assures that battery identifiers are globally unique, but I assume that the id (possibly combined with part number) makes it posssible to identify exactly which battery which is mounted.

I will double check with my colleagues and propose an update.

Reply:
Added a comment to clarify that it concerns a "unique" identifier. But the level of uniqueness is up to the OEM.